server/views/manage/premises/placements/show.njk

```diff
@@ -41,27 +44,57 @@
         },
         menus: PlacementUtils.actions(placement, user)
     }) }}
+    {{ mojSubNavigation({
+        label: 'Sub navigation',
+        items: tabItems
+    }) }}
+    {% if activeTab=='timeline' %}
+        <h2 class="govuk-heading-m">Placement timeline</h2>
+        {{applicationTimeline(timelineEvents)}}
+    {% elseif activeTab==='application' %}
+        {{ applicationReadonlyView(application) }}
+    {% elseif activeTab==='assessment' %}
+      {% for section in SummaryListUtils.summaryListSections(assessment, false) %}
+        <h2 class="govuk-heading-l">{{ section.title }}</h3>
+        {% for task in section.tasks %}
+          {{
+            govukSummaryList(task)
+          }}
+        {% endfor %}
+      {% endfor %}
+    {% elseif activeTab==='placementrequest' %}
+        {{ govukSummaryList({
+            attributes: {
+            'data-cy-section': "placement-request-summary"
+            },
+            rows:PlacementRequestUtils.adminSummary(placementRequestDetail).rows
+            }) }}
```

Comment:
Formatting is a bit off on that

Reply:
It is indeed.
Do you have an Intelij formatter/highlighter for nunjucks. I've not found one that works. I've noticed you sometimes 'prettify' templates so I'm guessing you have.

Reply:
Heh - you use Twig - I saw you left a review on the Twig plugin site!